gravitee-kubernetes-client/src/main/java/io/gravitee/kubernetes/client/KubernetesClientV1Impl.java

```diff
@@ -305,6 +318,7 @@ public KubernetesClientV1Impl(Vertx vertx, KubernetesConfig kubernetesConfig) {
                                 return Observable.just(response.toJsonObject().mapTo(type));
                             }
                         )
+                        .skip(1) // each time you connect to the API server, you get an initial ADD event representing the current resource
                         .doOnNext(emitter::onNext)
                         .doOnError(
```

Comment:
@jhaeyaert Do you think we need to add some sort of retry functionality if the case of error? for example trying to re-connect after few seconds for few times?

Reply:
Yes could be a good thing to do indeed 👍